checkov/sca_package_2/output.py

```diff
@@ -158,7 +172,8 @@ def create_cli_output(fixable: bool = True, *cve_records: list[Record]) -> str:
                             "package_version": package_version,
                             "lines": lines,
                             "root_package_lines": root_package_lines,
-                            "is_private_fix": record.vulnerability_details.get("is_private_fix")
+                            "is_private_fix": record.vulnerability_details.get("is_private_fix"),
+                            "reachability_risk_factors": reachability_risk_factors_tmp
```

Comment:
make sure that in case the condition in line 157 is fasle, you will have error here, as `reachability_risk_factors_tmp` will not be defined

Reply:
You are right